svthalia/concrexit
# Committee chair permission

Closes #3303.



### Summary
Added the option to add permissions that only a chair should have to MemberGroups and for  MemberGroupMemberships to have those chair permissions. Also for all existing MemberGroupMemberships that are already marked as chair also automatically have chair permissions 

### How to test
Steps to test the changes you made:
1. Go to a Committee or Society in the admin, select the desired chair-permissions in the respective field and save them
2. Go to the MemberGroupMembership admin and click on a MemberGroupMembership that you want to give chair permissions, mark the checkmark has_chair_permissions and save
3. Make sure that the member connected to that MemberGroupMembership has no other permissions form other MemberGroups
4. Log into that user, go to the admin page and try do the things that you gave permissions to do. It should work then
5. Log into a user that is part of that same MemberGroup and where his Membership does not have those permissions
6. Try to do the same actions as in 4. This time it should fail
